src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,20 @@
+const ADD_BOOK = 'books/booksSlice/ADD_BOOK';
+const REMOVE_BOOK = 'books/booksSlice/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addBook = (payload) => ({ type: ADD_BOOK, payload });
+export const removeBook = (payload) => ({ type: REMOVE_BOOK, payload });
+
+const booksReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'ADD_BOOK':
+      return [...state, action.payload];
+    case 'REMOVE_BOOK':
+      return state.filter((book) => book.id !== action.payload);
+    default:
+      return state;
+  }
+};
```

Comment:
- **[OPTIONAL]** Your books' actions and reducer should be set up using createSlice from the redux toolkit. This is a prerequisite for this project, but since the previous reviewer missed I will mark it as optional.

Reply:
Ohh okay
Thanks, @DammyShittu 
Can I do and you can give a personal review?